aries_vcx_core/src/indy/ledger/pool.rs

```diff
@@ -69,7 +69,11 @@ pub mod test_utils {
 
     use super::*;
 
-    pub fn get_temp_dir_path(filename: &str) -> PathBuf {
+    pub fn get_temp_dir_path() -> PathBuf {
```

Comment:
It seems like we have this function defined twice, it would be nice (and should be easy) to remove the duplication.

Reply:
it was duplicated before, but actually this PR removed the duplication